bin/instant-workstation

```diff
@@ -137,7 +147,39 @@ then
 fi
 if test -n "$sysctl"
 then
-	command="$command ; /usr/sbin/sysrc -f /etc/sysctl.conf $sysctl";
+	command="$command ; /sbin/sysctl -f /etc/sysctl.conf $sysctl";
```

Comment:
This doesn't seem right. sysrc(8) edits rc-files, which in this case means write-to-sysctl.conf (which is one of the files in that style). Changing to sysctl(8) now applies the changes immediately to the system, but doesn't save them for later.

Reply:
Hmm, interesting. When I run the script with KDE enabled, I get an error message `sysrc: net.local.stream.recvspace: name contains characters not allowed in shell`. I settled on the `sysctl` approach following the guidance at https://forums.freebsd.org/threads/sysrc-name-contains-characters-not-allowed-in-shell.72118/ but will do some more investigation.